monai/transforms/regularization/array.py

```diff
@@ -80,11 +85,20 @@ def apply(self, data: torch.Tensor):
         mixweight = weight[(Ellipsis,) + (None,) * len(dims)]
         return mixweight * data + (1 - mixweight) * data[perm, ...]
 
-    def __call__(self, data: torch.Tensor, labels: torch.Tensor | None = None):
-        self.randomize()
+    def __call__(self, data: torch.Tensor, labels: torch.Tensor | None = None, randomize=True):
+        data = convert_to_tensor(data, track_meta=get_track_meta())
+        data_t = convert_to_tensor(data, track_meta=False)
+        if labels is not None:
+            labels_t = convert_to_tensor(labels, track_meta=get_track_meta())
+            labels_t = convert_to_tensor(labels, track_meta=False)
```

Comment:
```suggestion
        data_t = convert_to_tensor(data, track_meta=get_track_meta())
        if labels is not None:
            labels_t = convert_to_tensor(labels, track_meta=get_track_meta())
```
Unless I missed something these lines were redundant?

Reply:
It was probably written this way in the beginning to avoid having a lot of meta data constantly recorded in the computation. It's fine to move here.